perun-core/src/main/java/cz/metacentrum/perun/core/implApi/ConsentsManagerImplApi.java

```diff
@@ -1,10 +1,33 @@
 package cz.metacentrum.perun.core.implApi;
 
+import cz.metacentrum.perun.core.api.Consent;
+import cz.metacentrum.perun.core.api.PerunSession;
+import cz.metacentrum.perun.core.api.exceptions.ConsentExistsException;
+import cz.metacentrum.perun.core.api.exceptions.ConsentNotExistsException;
+
 /**
  * Consents database logic.
  *
  * @author Radoslav Čerhák <r.cerhak@gmail.com>
  */
 public interface ConsentsManagerImplApi {
 
+	/**
+	 * Save consent to database.
+	 *
+	 * @param perunSession PerunSession
+	 * @param consent Consent
+	 * @return created consent
+	 * @throws cz.metacentrum.perun.core.api.exceptions.IllegalArgumentException if consent is null
```

Comment:
```suggestion
	 * @throws ConsentExistsException if consent exists
```
isn't InternalErrorException thrown when consent is null? please fix the javadoc to match the actual exceptions that are thrown. you won't need to use the whole package name, just the exception name is okay.